packages/auth-remix/src/client.ts

```diff
@@ -27,8 +27,8 @@ export class RemixClientAuth {
   /** @internal */
   constructor(options: RemixAuthOptions) {
     this.options = {
-      authCookieName: "edgedb-session",
-      pkceVerifierCookieName: "edgedb-pkce-verifier",
+      authCookieName: "gel-session",
+      pkceVerifierCookieName: "gel-pkce-verifier",
```

Comment:
Hmm.. I wonder if we can support a fallback here so that apps that update to use the latest packages don't accidentally log all of their users out.

Let's think of an approach here that will work easily for new users and upgrading users.

Reply:
oh yes, this is also a thing with all other auth libs

Reply:
hey, I pushed a commit (I can maybe make some code more neat), but the functionality is basically:
in constructor we define gel (can be overriden by user names),
when I parse I parse both (firstly gel and if not exists check for edgedb one),
when I delete cookies I delete both gel and edgedb,
when I create new cookies I create only gel...